specification/entities.yaml

```diff
@@ -72,6 +72,16 @@ components:
       required:
         - data_source_oddrn
 
+    RelationshipList:
+      type: object
+      properties:
+        items:
```

Comment:
Why don't we have `data_source_oddrn ` property like in DataEntityList? As I understand it is needed for API to know in which Data Source to ingest the relationships.

Reply:
there is no need in this, because we will receive oddrns and name from BaseObject in Relationship. In case we will add it there it be will be duplicate. All dataset related oddrns will be in also in Relationship

Reply:
done